src/components/FisheriesReport.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="report">
+    <h1 class="title is-1">{{ selectedGroup }} in {{ selectedRegion }}</h1>
+    <div id="report" v-if="groupedFisheries[selectedRegion] != undefined">
+      <div
+        v-for="fishery in groupedFisheries[selectedRegion][selectedGroup]"
+        :key="fishery"
+      >
+        <h3 class="title is-3" v-html="fishery['name']"></h3>
+        <table>
+          <tr>
+            <td style="width: 200px">Access</td>
+            <td v-html="fishery['entry']"></td>
+          </tr>
+          <tr>
+            <td>Species</td>
+            <td v-html="fishery['species']"></td>
+          </tr>
+          <tr>
+            <td>Gear</td>
+            <td v-html="fishery['gear']"></td>
+          </tr>
+          <tr>
+            <td>Region</td>
+            <td v-html="fishery['region']"></td>
+          </tr>
+          <tr>
+            <td>CFEC code</td>
```

Comment:
We haven't asked them about this, but they are using the label "CFEC code" in the previous version of their app, so I recommend using this unless they tell us otherwise.